src/main/java/com/plaid/client/model/paymentinitiation/Bacs.java

```diff
@@ -0,0 +1,28 @@
+package com.plaid.client.model.paymentinitiation;
+
+public class Bacs {
```

Comment:
Do we expect the setter methods (`setAccount` and `setSortCode`) for Bacs to be used? IIRC we don't include methods in the public plaid-java API by default if we don't expect usage.

Reply:
⚡